plugin/godebug.vim

```diff
@@ -64,12 +64,12 @@ endfunction
 
 function! godebug#debug(bang, ...) abort
   call godebug#writeBreakpointsFile()
-  return go#term#new(a:bang, ["dlv", "debug", "--init=" . g:godebug_breakpoints_file])
+  return go#term#new(a:bang, ["dlv", "debug", "--init=" . g:godebug_breakpoints_file], "%-G#\ %.%#")
```

Comment:
@joaoofreitas What `"%-G#\ %.%#"` is doing?

Reply:
Is the new vim-go required parameter for calling the function `go#term#new()`. 

Since you wrote the script. vim-go got an update for an error formatting option on launch of a new terminal.
Since then your script has been giving an error of _"not enough parameters"_.

```vim
function! go#term#new(bang, cmd, errorformat) abort
  return go#term#newmode(a:bang, a:cmd, a:errorformat, go#config#TermMode())
endfunction
```


Answering your question the `"%-G#\ %.%#"` is a errorformat pattern that parses over the source code line. And prints it as a whole message.

This error format was proposed by @dude333 on the [issue](https://github.com/jodosha/vim-godebug/issues/17)#17.